src/modules/interface/p2pDTR/p2p_interface.h

```diff
@@ -0,0 +1,47 @@
+/**
+ * ,---------,       ____  _ __
+ * |  ,-^-,  |      / __ )(_) /_______________ _____  ___
+ * | (  O  ) |     / __  / / __/ ___/ ___/ __ `/_  / / _ \
+ * | / ,--´  |    / /_/ / / /_/ /__/ /  / /_/ / / /_/  __/
+ *    +------`   /_____/_/\__/\___/_/   \__,_/ /___/\___/
+ *
+ * Crazyflie control firmware
+ *
+ * Copyright (C) 2019 Bitcraze AB
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, in version 3.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * p2p_interface.h
+ * 
+ */
+
+#ifndef _P2P_INTERFACE_H_
+#define _P2P_INTERFACE_H_
+
+#include "FreeRTOS.h"
+#include "DTR_types.h"
+#include "radiolink.h"
+#include "token_ring.h"
+
+#define INCOMING_DTR_QUEUE_SIZE 10
+
+// Broadcasts a DTR packet through the P2P network
+void DTRsendP2Ppacket(const DTRpacket* packet) ;
+
+// Puts the DTR packet in the queue for the token ring to pick up.
+void DTRp2pIncomingHandler(P2PPacket *p);
```

Comment:
This function should probably return a bool to indicate if the packet was handled or not. This will allow a user to know if any other action is required.

Reply:
I changed it into returning true if it was handled from the protocol and false otherwise.